bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptions.java

```diff
@@ -208,9 +204,9 @@ private BigtableOptions(
         !Strings.isNullOrEmpty(zone), "Zone must not be empty or null.");
     Preconditions.checkArgument(
         !Strings.isNullOrEmpty(cluster), "Cluster must not be empty or null.");
-    this.adminHost = Preconditions.checkNotNull(adminHost);
-    this.clusterAdminHost = clusterAdminHost;
-    this.host = Preconditions.checkNotNull(host);
+    this.tableAdminHost = Preconditions.checkNotNull(tableAdminHost);
+    this.clusterAdminHost = Preconditions.checkNotNull(clusterAdminHost);
```

Comment:
It's ok for clusterAdminHost to be null.  In fact, most users will not set this.